### DISCUSSION TITLE
What server specs are recommended / common for running Fider on a VPS? (RAM / CPU Cores)

### DISCUSSION THREAD

**Discussion by spencerflagg** [getfider/fider#1062] [Q&A]
How much memory and how many cores are recommended?
What specs are people running on right now?

Mine is 1 GB Ram / 1 CPU Core; Linode VPS

**Comment by boydthomson** [comment]
I'm running mine on half that RAM and it hasn't gone > 128MB usage, but I've only got a few test users thus far.  You've got a db server and a web server... it's going to depend on # simultaneous users you need to support.
